### DISCUSSION TITLE
My Code is not working - Hashtable error?

### DISCUSSION THREAD

**Discussion by jart2021** [PowerShell/PowerShell#24045] [Q&A]
This is my code, to get quotas of onedrive. 
 
```
# Import users from  CSV file
$usuarios = Import-Csv "C:\Temp\lista1.csv"  

$resultados = @()

foreach ($usuario in $usuarios) {
    try {
        # get OneDrive quota by user
        
        $cuota = Get-PnPUserOneDriveQuota -Account $usuario.username

        # Add Email & Quota to results 
        $resultados += [PSCustomObject]@{
            userEmail = $usuario.username
            cuotaMB = $cuota.StorageQuota / 1MB
        }
    } catch {
        # Manage any excepción 
        Write-Warning "Error al obtener la cuota para $usuario"
    }
}

# Export results to a CSV file
$resultados | Export-Csv -Path "onedrive_cuotas.csv" 

Write-Host "Cuotas de OneDrive exportadas a onedrive_cuotas.csv"
```

but i got this error msg

_WARNING: Error al obtener la cuota para @{IsReadOnly=False; IsFixedSize=False; IsSynchronized=False; Keys=System.Collections.Hashtable+KeyCollection; Values=System.Collections.Hashtable+ValueCollection; SyncRoot=System.Object; Count=2}
WARNING: Error al obtener la cuota para @{IsReadOnly=False; IsFixedSize=False; IsSynchronized=False; Keys=System.Collections.Hashtable+KeyCollection; Values=System.Collections.Hashtable+ValueCollection; SyncRoot=System.Object; Count=2}

Cuotas de OneDrive exportadas a onedrive_cuotas.csv_

Could anyone help me please?

**Comment by rhubarb-geek-nz** [comment]
> but i got this error msg

The error message is _your_ error message.

I suggest you look at $PSItem within the catch to see the actual error.
